src/Microsoft.AspNetCore.Mvc.ViewFeatures/Rendering/TagBuilder.cs

```diff
@@ -298,37 +298,77 @@ public void WriteTo(TextWriter writer, HtmlEncoder encoder)
                 throw new ArgumentNullException(nameof(encoder));
             }
 
-            switch (TagRenderMode)
+            WriteTo(this, writer, encoder, TagRenderMode);
+        }
+
+        /// <summary>
+        /// Returns an <see cref="IHtmlContent"/> that renders the body 
+        /// for the given <see cref="TagBuilder"/>.
```

Comment:
Sorry I missed this before. There is no "given" `TagBuilder`; should probably say just `that renders the body` or maybe `that renders the body of this <see cref="TagBuilder"/>`. Same for `<returns>` and the methods just below.